### PR TITLE
Name to ID

### PR DESCRIPTION
**Pull request by colincoleman**

### CODE REVIEW

File: outputs.tf [hunk 1]
```diff
@@ -11,9 +11,9 @@ output "name" {
   value       = "${var.name_prefix}"
 }
 
-output "role_name" {
+output "role_id" {
   description = "The name of the execution role."
```
Comment (mikael-lindstrom) [inline outputs.tf]: If this now is the id the description should reflect this.

**Review by mikael-lindstrom — CHANGES_REQUESTED** [review]
Won't this break backwards compatibility if people are using the `role_name` now?

**Comment by colincoleman** [comment]
This change is to address this issue:
telia-oss/terraform-aws-ecs#7

File: outputs.tf [hunk 2]
```diff
@@ -11,9 +11,9 @@ output "name" {
   value       = "${var.name_prefix}"
 }
 
-output "role_name" {
-  description = "The name of the execution role."
-  value       = "${aws_iam_role.main.name}"
+output "role_id" {
+  description = "The id of the execution role."
+  value       = "${aws_iam_role.main.id}"
```
Comment (mikael-lindstrom) [inline outputs.tf:16]: Shouldn't this be `unique_id`?

**Pull request closed without merging**